js/script.js

```diff
@@ -0,0 +1,250 @@
+const createList = (items, className) => {
+  const ul = createElement('ul', className);
+  items.forEach((item) => {
+    const li = createElement('li', `${className}-item`);
+    li.innerText = item;
+    ul.appendChild(li);
+  });
+  return ul;
+};
+
+// fuction to create button element given the class name, id and text content of the button
+const createButton = (className, id, text) => {
+  const btn = createElement('button', className);
+  btn.id = id;
+  btn.innerText = text;
+  return btn;
+};
+
+const projectContainer = document.getElementById('projectContainer');
+
+const createtopProjectCon = (project1) => {
+  const topProjectEl = createElement('div', project1);
+
+  const project1Image = createElement('div', 'project1Image');
+  topProjectEl.appendChild(project1Image);
+
+  const image1 = createElement('img', 'image1');
+  image1.src = topProjectObj.image;
+  project1Image.appendChild(image1);
+
+  const project1Info = createElement('div', 'project1Info');
+  topProjectEl.appendChild(project1Info);
+
+  const project1Title = createElement('h3', 'project1Title');
+  project1Title.innerHTML = topProjectObj.title;
+  project1Info.appendChild(project1Title);
+
+  const project1Description = createElement('p', 'project1Description');
+  project1Description.innerText = topProjectObj.description;
+  project1Info.appendChild(project1Description);
+
+  const project1Languages = createList(topProjectObj.technologies, 'project1Languages');
+  project1Info.appendChild(project1Languages);
+
+  const project1LinkBtn = createButton('popupBtn', topProjectObj.id, 'See Project');
+  project1Info.appendChild(project1LinkBtn);
+
+  return topProjectEl;
+};
+
+const createOtherProject = (otherProject) => {
+  const otherProjectEl = createElement('div', 'otherProject');
+
+  const otherProjectInfo = createElement('div', 'otherProjectInfo');
+  otherProjectEl.appendChild(otherProjectInfo);
+
+  const otherProjectTitle = createElement('h3', 'otherProjectTitle');
+  otherProjectTitle.innerHTML = otherProject.title;
+  otherProjectInfo.appendChild(otherProjectTitle);
+
+  const otherProjectDescription = createElement('p', 'otherProjectDescription');
+  otherProjectDescription.innerText = otherProject.description;
+  otherProjectInfo.appendChild(otherProjectDescription);
+
+  // const otherProjectLanguages = createList(projectArr.technologies, 'otherProjectLanguages');
+  // otherProjectInfo.appendChild(otherProjectLanguages);
+
+  const otherProjectLink = createElement('div', 'otherProjectLink');
+  otherProjectEl.appendChild(otherProjectLink);
+
+  const otherProjectLinkBtn = createButton('popupBtn', otherProject.id, 'See Project');
+  otherProjectLink.appendChild(otherProjectLinkBtn);
+
+  return otherProjectEl;
+};
+
+const popupTopProject = (projectPopup) => {
+  const popupTopProjectEl = createElement('div', projectPopup);
+
+  const popupHeader = createElement('div', 'popupHeader');
```

Comment:
- [ ] Hello Team, Good job you have done, But currently I could not open the popup to see the `projects popup window`. When the user clicks (or taps) the `See Project` button is clicked, the popup with details about the project is expected to appear. Please make sure the following points....
-  When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- The popup is responsive, it fits mobile and desktop sizes properly.
- The popup follows the design for both desktop and mobile versions.
> I have checked for both the live and clone versions and it's not working. Please make sure it's working fine before the next review request.